fhir_river_api/src/app.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+import os
+from flask import Flask
+from flask import request
+from flask_restful import Api
+
+from fhir_river_api.src.config.logger import create_logger
+from fhir_river_api.src.errors import OperationOutcome
+from fhir_river_api.src.producer_class import RiverApiProducer
+import time
+
+logger = create_logger("fhir_river_api")
+
+# Create flask app object
+app = Flask(__name__)
+api = Api(app)
+
+producer = RiverApiProducer(broker=os.getenv("KAFKA_BOOTSTRAP_SERVERS"))
+
+
+@app.route("/run_sample", methods=["POST"])
+def trigger_sample_extractor():
+    """
+    Extract all records for the specified resource
```

Comment:
It's the wrong docstring here. It was `Extract record for the specified resource and primary key value` previously

Reply:
Indeed!